### PR TITLE
bump to 1.8.0-rc3

### PR DESCRIPTION
**Pull request by imikushin**
bump to 1.8.0-rc3

also, build on OS X

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-base-image/build
-base-image/cache
+base-image
```
Comment (ibuildthecloud) [inline .dockerignore]: we don't want to ignore all of base-image, just build and cache

File: scripts/Dockerfile.build [hunk 2]
```diff
@@ -6,3 +6,7 @@ RUN go get github.com/tools/godep
 ENV GOPATH /go/src/github.com/rancher/docker-from-scratch/Godeps/_workspace:/go
 
 WORKDIR /go/src/github.com/rancher/docker-from-scratch
+
+COPY . .
```
Comment (ibuildthecloud) [inline scripts/Dockerfile.build]: argh...  this copies all of base-image.  I see why you ignored it

File: scripts/package [hunk 3]
```diff
@@ -5,7 +5,11 @@ cd $(dirname $0)/..
 
 . ./scripts/common
 
-docker run --rm -v `pwd`:/go/src/github.com/rancher/docker-from-scratch dockerscratch-build strip --strip-all build/dockerlaunch
+ID=$(docker run -d dockerscratch-build strip --strip-all build/dockerlaunch)
```
Comment (ibuildthecloud) [inline scripts/package]: Can you move this to build?  I'd like the result of ./script/build be that I get a dockerlaunch binary local on my host

Comment (imikushin) [inline scripts/package]: Done

**Pull request merged**